Comments/ClientApp/src/context/UserContext.js

```diff
@@ -130,17 +130,6 @@ export class UserProvider extends React.Component<PropsType, StateType> {
 			);
 	}
 
-	getFeatureFlags = async () => {
```

Comment:
could this code be reinstated and the state.organisationalCommentingFeature variable be removed please?